### PR TITLE
New Accordion Component

### PR DESCRIPTION
**Pull request by daviddkkim**
Closes #

### Changes

// Describe what you changed
Added an accordion component to be used in tokens project and more.
It can expand and collapse to show or hide content.

In the storybook, you will see that clicking the toggle propagates the event to the accordion expansion as well. This can be stopped by adding stopPropagation to the container of the toggle and the label. 

### Screenshots
https://user-images.githubusercontent.com/12561526/126571203-e1612225-9560-48ac-b799-dfb0dc9d4821.mp4

// Add screenshots here if relevant

### Checklist
Check all that apply

- [ ] Updated documentation to reflect changes
- [ ] Added entry to top of Changelog with link to PR (not issue)
- [ ] Tests pass
- [ ] Peer reviewed and approved
- [ ] Signed [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: src/Components/Accordion/Accordion.scss [hunk 1]
```diff
@@ -0,0 +1,152 @@
+@import '../../Styles/variables';
+
+.cf-accordion {
+  width: 100%;
+  margin: 0;
+  padding: 0;
+  margin-bottom: 2px;
```
Comment (mavarius) [inline src/Components/Accordion/Accordion.scss:7]: This should be relative to one of our margin variables.

File: src/Types/index.tsx [hunk 2]
```diff
@@ -159,6 +159,11 @@ export enum DropdownItemType {
   None = 'none',
 }
 
+export enum IconPlacement {
```
Comment (mavarius) [inline src/Types/index.tsx:162]: I think we have some duplication happening here. `IconPlacement` and `Direction` are essentially just `left` and `right`, and `Alignment` is `left`, `right`, and `center`. Perhaps we can consolidate these.

Also, `Orientation` and `ComponentOrientation` are just `horizontal` and `vertical`.

**Review by mavarius — APPROVED** [review]
Looks good, just a few comments.

File: src/Types/index.tsx (hunk 2 above)
Comment (daviddkkim) [reply src/Types/index.tsx]: got rid of icon placmeent for direction !

File: src/Components/Accordion/Accordion.scss (hunk 1 above)
Comment (daviddkkim) [reply src/Components/Accordion/Accordion.scss]: done!

**Pull request merged**